backend/s3/s3.go

```diff
@@ -269,10 +269,21 @@ func init() {
 				Value: "s3.tor01.objectstorage.service.networklayer.com",
 				Help:  "Toronto Single Site Private Endpoint",
 			}},
+		}, {
+			Name:     "endpoint",
+			Help:     "Endpoint for Wasabi S3 API.\nRequired when using Wasabi.",
+			Provider: "Wasabi",
+			Examples: []fs.OptionExample{{
+				Value:    "s3.wasabisys.com",
```

Comment:
Just wondering why you duplicated the "endpoint" setting - I think stuffing it into the existing one would work just as well wouldn't it?  When the config runs it would just give the Wasabi endpoints

Reply:
I was following the style of the IBM COS section, but that's a much longer list.  Should have looked closer at the Digital Ocean case which is a better match.  I'll change it and push a new version.